Makefile

```diff
@@ -170,6 +170,19 @@ release: static gen-device
 	./build/tinygo build-builtins -target=armv7em-none-eabi -o build/release/tinygo/pkg/armv7em-none-eabi/compiler-rt.a
 	tar -czf build/release.tar.gz -C build/release tinygo
 
+# Override this to match the version of software being released.
+RELEASE_VERSION ?= 1.0.0
+
+# Override this to match the arch for software being released.
+RELEASE_ARCH ?= amd64
```

Comment:
You can use `$(shell dpkg --print-architecture)` for this.

Reply:
I assumed that we would probably have to download the file from the CI server, build the DEB, and then upload it to Github, so whoever does this would set this per release. Very possible that the machine doing the packaging not same as machine doing the building.